floris/turbine_library/nrel_5MW.yaml

```diff
@@ -39,6 +39,12 @@ power_thrust_table:
   cosine_loss_exponent_tilt: 1.88
   # Cosine exponent for power loss due to yaw misalignment.
   cosine_loss_exponent_yaw: 1.88
+  # Helix parameters
+  helix_a: 1.802
+  helix_power_b: 4.568e-03
+  helix_power_c: 1.629e-10
```

Comment:
This is a very small number - is the helix model sensitive enough to changes at that order of magnitude? In other words, could this be 0?

Reply:
I agree that it's small, but it is multiplied by `base_powers`, which is in the order of 1e6. As a result, this sum is more or less the same magnitude as `helix_power_b`.